packages/mui-material/src/Autocomplete/Autocomplete.d.ts

```diff
@@ -81,6 +81,7 @@ export interface AutocompleteRenderInputParams {
     className: string;
     startAdornment: React.ReactNode;
     endAdornment: React.ReactNode;
+    onMouseDown: (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => void;
```

Comment:
`onMouseDown` is passed to renderInput args, but type doesn't refelct that

https://github.com/mui/material-ui/blob/141fc36df4aa6ab496b2034e4401cd3f41c29b1b/packages/mui-material/src/Autocomplete/Autocomplete.js#L741